_rules/zoom-text-no-overflow-clipping-59br37.md

```diff
@@ -112,6 +114,27 @@ This [text node][] is restricted to a single line, by setting a `line-height` th
 
 #### Passed Example 4
 
+Assuming a [used][] [line-height][] of 1.2 when `normal`, this [text node][] is restricted to a single line. A link to a full version of the poem is also provided.
+
+```html
+<style>
+	.wordClip {
+		overflow: hidden;
+		word-wrap: break-word;
+		height: 18px;
+		line-height: normal;
```

Comment:
Ah, that's a good point... I'd be fine with removing the example altogether given that it could cause consistency issues.

Reply:
I agree that it's best to leave passed example 4 out.